src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      data-qa="big"
+      name="smallform"
+    >
+      <label for="searchbig">
+        <img
+          src="./images/search.svg"
+          alt="searchicon"
```

Comment:
Це буде читати скрінрідер, тому краще робити відступи між словами
```suggestion
          alt="search icon"
```